src/racer/nameres.rs

```diff
@@ -11,13 +11,9 @@ use matchers::find_doc;
 use cargo;
 use std::path::{Path, PathBuf};
 use std::{self, vec};
+use std::iter;
```

Comment:
Can you combine this with the `use` on the previous line?

Reply:
Aye.